common/ts/package.json

```diff
@@ -5,12 +5,11 @@
   "author": "SonarSource",
   "license": "LGPL-3.0",
   "dependencies": {
-    "azure-devops-node-api": "^9.0.1",
-    "azure-pipelines-task-lib": "^2.8.0",
+    "axios": "0.27.2",
```

Comment:
Why use axios version `0.27.2` rather than a more recent one? [Latest seems to be 1.5.0](https://www.npmjs.com/package/axios)

Reply:
I tried to use the latest axios version, the problem is that they use modern esmodule syntax in their library and the build system of this project is not fit for that. I had a lot of errors because of that, and I tried to fix them, but I was going down a big rabbit hole and I felt it was too much work.

Reply:
https://stackoverflow.com/questions/73958968/cannot-use-import-statement-outside-a-module-with-axios

This link gives more info about that

Reply:
I hope it doesn't introduce a different set of vulnerabilities 😅